resources/qml/MachineSettings/GcodeTextArea.qml

```diff
@@ -70,6 +71,7 @@ UM.TooltipArea
             selectionColor: UM.Theme.getColor("text_selection")
             selectedTextColor: UM.Theme.getColor("text")
             wrapMode: TextEdit.NoWrap
+            padding: -border.width //Take care of the border width to fit the text.
```

Comment:
border here is out of scope. creates qml warnings. perhaps can use the borders from rectangle below as : rectid.border

Reply:
Oh yes indeed, I just move the code content from below but didn't check the console.. Visually it looked good :laughing: